src/app/services/new-studing-year.service.ts

```diff
@@ -15,9 +15,9 @@ export class NewStudingYearService {
     })
   };
 
-  getGroups(): Observable<Group[]> {
+  getGroups(): Observable<NewGroup[]> {
```

Comment:
why do we have a different model for a new group?

Reply:
I'll fix it

Reply:
Fixed